test/unittest/tests/block/test_extent_list_insert_wo_block.cpp

```diff
@@ -0,0 +1,197 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * block_ext.c: [extent_list2] Test extent list functions part 2. (More to come.)
+ *
+ * Test insert functions without block: __block_ext_insert, and __block_off_insert.
+ */
+
+/* Choose one */
```

Comment:
From what I understand, we should only be using `/* */` for comments greater than two lines.

Reply:
Done. New line 15 was deleted in response to an earlier review comment.